src/modules/webRTC/components/ChatUserPage.tsx

```diff
@@ -0,0 +1,35 @@
+import { useCallback, useRef } from "react";
+import { Box } from "../../components/Box";
+import { Input } from "../../components/Input";
+import { T } from "../../components/Text";
+
+export type UserInfo = {
+    name: string,
+    //... other user info
+}
+
+export const ChatUserPage = ({ userInfo, setUserInfo }: { userInfo: UserInfo, setUserInfo: React.Dispatch<UserInfo> }) => {
+
+    const chatUserInputRef = useRef<HTMLInputElement>(null);
+
+    const getUserData = useCallback((): UserInfo => {
```

Comment:
handling parents state with ref is uhh, very clunky, this could be context, but state manager is a better solution